### PR TITLE
Enable more strategy stages

### PR DESCRIPTION
**Pull request by tomaslin**

### CODE REVIEW

**Comment by tomaslin** [comment]
depends on https://github.com/spinnaker/orca/pull/677

**Comment by tomaslin** [comment]
@anotherchrisberry @zanthrash  ping

File: app/scripts/modules/core/pipeline/config/preconditions/preconditionList.directive.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
         </td>
         <td>
           <a class="btn btn-xs btn-link" href
-             ng-click="preconditionListCtrl.editPrecondition(precondition)">Edit</a>
+             ng-click="preconditionListCtrl.editPrecondition(precondition,strategy)">Edit</a>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/preconditions/preconditionList.directive.html]: should add a space between arguments, e.g. `(precondition, strategy)`

File: app/scripts/modules/core/pipeline/config/preconditions/types/clusterSize/additionalFields.html [hunk 2]
```diff
@@ -1,17 +1,8 @@
+
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/preconditions/types/clusterSize/additionalFields.html]: remove

File: app/scripts/modules/core/pipeline/config/preconditions/types/clusterSize/additionalFields.html [hunk 3]
```diff
@@ -1,17 +1,8 @@
...
-             name="cluster"
-             class="form-control input-sm "
-             ng-model="precondition.context.cluster"
-             required>
-      </input>
-    </div>
-  </div>
-  <div class="form-group row">
-    <div class="col-sm-3 sm-label-left">Accounts</div>
+    <div class="col-sm-3 sm-label-left">Account</div>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/preconditions/types/clusterSize/additionalFields.html]: Good catch

File: app/scripts/modules/core/pipeline/config/preconditions/types/clusterSize/additionalFields.html [hunk 4]
```diff
@@ -32,6 +23,18 @@
...
+      <div class="col-sm-9">
+        <input type="text"
+               name="cluster"
+               class="form-control input-sm "
+               ng-model="precondition.context.cluster"
+               required>
+        </input>
+      </div>
+    </div>
+  </div>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/preconditions/types/clusterSize/additionalFields.html]: :+1:

File: app/scripts/modules/core/pipeline/config/stages/modifyScalingProcess/modifyScalingProcessStage.html [hunk 5]
```diff
@@ -23,10 +23,7 @@
     </select>
   </stage-config-field>
   <stage-config-field label="Processes" field-columns="8">
-    <p ng-if="!stage.credentials">
-      Please select an account to pick processes.
-    </p>
-    <div ng-if="stage.credentials" class="checkbox" style="padding-left:0;">
+    <div class="checkbox" style="padding-left:0;">
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/stages/modifyScalingProcess/modifyScalingProcessStage.html]: :+1:

File: app/scripts/modules/core/pipeline/config/stages/resizeAsg/aws/resizeAsgStage.html [hunk 6]
```diff
@@ -17,6 +17,7 @@ <h4 class="text-center">
         <input type="text" required ng-model="stage.cluster" class="form-control input-sm" />
       </stage-config-field>
     </div>
+    </div>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/stages/resizeAsg/aws/resizeAsgStage.html]: This seems either unnecessary or very necessary, in which case the indentation is off on the preceding lines.

File: app/scripts/modules/core/pipeline/config/stages/script/scriptStage.html [hunk 7]
```diff
@@ -32,6 +33,7 @@ <h4 ng-if="viewState.loading" class="text-center">
     <input type="text" class="form-control input-sm"
            ng-model="stage.cluster"/>
   </stage-config-field>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/stages/script/scriptStage.html]: should indent these lines

**Comment by anotherchrisberry** [comment]
a bunch of very little formatting (and one potentially larger formatting) things; otherwise, LGTM.

**Pull request merged**